crates/iceberg-catalog/src/catalog/views/create.rs

```diff
@@ -94,20 +94,21 @@ pub(crate) async fn create_view<C: Catalog, A: AuthZHandler, S: SecretStore>(
 
     let view_id: TabularIdentUuid = TabularIdentUuid::View(uuid::Uuid::now_v7());
 
-    let view_location = storage_profile.initial_tabular_location(namespace_id, view_id);
+    let namespace_location = storage_profile.default_namespace_location(namespace_id)?;
+    let view_location = storage_profile.default_tabular_location(&namespace_location, view_id);
     let mut request = request;
-    let metadata_location = storage_profile.initial_metadata_location(
+    let metadata_location = storage_profile.metadata_location(
```

Comment:
default_metadata_location instead?

Reply:
It was originally called metadata_location but we decided to rename to signal that it may be giving you something else than what's written into the db already, the `initial_` prefix caused some confusion in #156, i saw you used `default` for namespace location already, what do you think?